source/guide_mattermost.rst

```diff
@@ -160,12 +171,94 @@ When upgrading to Mattermost 6.4 or newer you need to change the collation of th
   [isabell@stardust ~]$ mysql -e "ALTER DATABASE isabell_mattermost COLLATE = utf8mb4_general_ci;"
   [isabell@stardust ~]$
 
+Automatic Update
+-----------------
+
+Use the script attached to update Mattermost to the current version. Run the script above the mattermost-root directory.
+
+.. code-block:: shell
+
+	#!/bin/sh
+	printf "\n===\nWelcome to updating Mattermost... let us begin!\n===\n"
+
+	printf "\n== Preflight Checks ==\n"
+	if [ ! -d "./mattermost" ]; then
+	printf "Error: ./mattermost directory does not exist. Are you running in the correct place?. Exit\n"
+	exit 1
+	else 
+		printf "Directory ./mattermost found.\n"
+	fi
+
+	database=$(cat ./mattermost/config/config.json | jq '.SqlSettings.DataSource' | grep -o '/.*?' | tail -c +2 | head -c -2)
+	if [ -z "$database" ] 
+	then
+		printf "\nError: Could not extract the database name from ./mattermost/config/config.json. Maybe the script runs in the wrong directory? Exit 1.\n"
+		exit 2
+	else 
+		printf "Using database: $database\n"
+	fi
+
+	version=$1
+	if [ -z "$version" ] 
+	then
+		printf "No manual version given. Check from official website ...\n"
+
+		version=$(curl -s https://api.github.com/repos/mattermost/mattermost-server/releases/latest | jq '.tag_name' | tail -c +3 | head -c -2)
```

Comment:
`head` and `tail` is magic you don't need this here.

```suggestion
		tag_name=$(curl -s https://api.github.com/repos/mattermost/mattermost-server/releases/latest | jq --raw-output '.tag_name')
		version=${TAG_NAME:1} # This will remove the first character `v` from the version tag `v1.2.3`
```